[2024-05-24T11:59:47.824+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-24T11:59:47.834+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2023-02-19T00:00:00+00:00 [queued]>
[2024-05-24T11:59:47.838+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2023-02-19T00:00:00+00:00 [queued]>
[2024-05-24T11:59:47.839+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-24T11:59:47.846+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_and_upload_to_s3> on 2023-02-19 00:00:00+00:00
[2024-05-24T11:59:47.848+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=191) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-24T11:59:47.849+0000] {standard_task_runner.py:63} INFO - Started process 198 to run task
[2024-05-24T11:59:47.850+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'upload_from_github_to_s3', 'download_and_upload_to_s3', 'scheduled__2023-02-19T00:00:00+00:00', '--job-id', '248', '--raw', '--subdir', 'DAGS_FOLDER/upload_from_github_to_s3_dag.py', '--cfg-path', '/tmp/tmpb0kwonbw']
[2024-05-24T11:59:47.850+0000] {standard_task_runner.py:91} INFO - Job 248: Subtask download_and_upload_to_s3
[2024-05-24T11:59:47.874+0000] {task_command.py:426} INFO - Running <TaskInstance: upload_from_github_to_s3.download_and_upload_to_s3 scheduled__2023-02-19T00:00:00+00:00 [running]> on host b2ab95bd73fd
[2024-05-24T11:59:47.914+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_from_github_to_s3' AIRFLOW_CTX_TASK_ID='download_and_upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2023-02-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-02-19T00:00:00+00:00'
[2024-05-24T11:59:47.915+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-24T11:59:49.343+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2024-05-24T11:59:49.415+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-24T11:59:49.415+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-24T11:59:49.419+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=upload_from_github_to_s3, task_id=download_and_upload_to_s3, run_id=scheduled__2023-02-19T00:00:00+00:00, execution_date=20230219T000000, start_date=20240524T115947, end_date=20240524T115949
[2024-05-24T11:59:49.469+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-24T11:59:49.477+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-24T11:59:49.478+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
